docs/classic-ui/images.md

```diff
@@ -0,0 +1,114 @@
+---
+html_meta:
+  "description": ""
+  "property=og:description": ""
+  "property=og:title": ""
+  "keywords": ""
+---
+
+(classic-ui-images-label)=
+
+# Image resolving and scaling
+
+Image scaling is done in plone.namedfile.scaling and most interesting the ImageScaling view.
```

Comment:
Where is "ImageScaling view"? What is it? This might be obvious to experienced Plone developers, but newbies have no idea.

Reply:
Precondition: A (Dexterity) content type with one or more image fields (`plone.namedfile.NamedBlobImage`). `context` is an instance of this content type.

View is accessed via traversal: 
`context.restrictedTraverse("@@images/FIELDNAME")` to get the view for s specific field.

The (by default registered) view class is `plone.namedfile.scaling.ImageScaling`.

The view is 
- on one hand a tool for programmers to get reliable and cachable URLs of scales of an image of above mentioned image field, 
- on the other hand it is used as a URL in the browser like `../my.jpg/@@images/SCALEUID` or (not cachable and not recommended) like `../my.jpg/@@images/FIELDNAME/SCALENAME`.

Reply:
I just found plone.api is the way to go, so forget about the traversal part. Only needed in plone core development where plone.api can not be used.

Reply:
restrictedTraverse is not working for this case, it's not able to resolve the image field and scale.

Reply:
I don't think I got an answer to my original question: Where is "ImageScaling view"? What is it?

Reply:
it's linked now